src/graph/ChcGraph.cc

```diff
@@ -6,6 +6,8 @@
 
 #include "ChcGraph.h"
 
+#include "../transformers/CommonUtils.h"
+#include "../TransformationUtils.h"
```

Comment:
```suggestion
#include "transformers/CommonUtils.h"
#include "TransformationUtils.h"
```

This should be OK, no?

Reply:
Fixed